t/Legacy/Regression/736_use_ok.t

```diff
@@ -18,13 +18,16 @@ sub capture(&) {
     return $warn || "";
 }
 
+{
+local $TODO = "known to fail on $]" if $] le "5.006002";
 my $file = __FILE__;
 my $line = __LINE__ + 2;
 like(
-    capture { use_ok 'MyWarner' },
+    capture { local $TODO; use_ok 'MyWarner' },
```

Comment:
This could also use a comment. Why is this necessary?

Reply:
I thought it is easy to understand reason... But I added explaining git fixup commit. It is better now?